examples/tracks_3d_with_graph.py

```diff
@@ -0,0 +1,46 @@
+import napari
+import numpy as np
+
+def _circle(r, theta):
+    x = r*np.cos(theta)
+    y = r*np.sin(theta)
+    return x, y
+
+
+def tracks_3d_merge_split():
+    """ create tracks with splitting and merging """
```

Comment:
Just for future reference, not necessary to fix in this PR, but this is giving my OCD a brain aneurysm. =P

```suggestion
    """Create tracks with splitting and merging."""
```